plugin/path_config.go

```diff
@@ -72,6 +72,11 @@ func (b *backend) configFields() map[string]*framework.FieldSchema {
 		Type:        framework.TypeString,
 		Description: `Text to insert the password into, ex. "customPrefix{{PASSWORD}}customSuffix".`,
 	}
+	fields["out_of_band_rotation_seconds"] = &framework.FieldSchema{
+		Type:        framework.TypeDurationSecond,
+		Description: "The number of seconds after a Vault rotation where, if Active Directory shows a later rotation, it should be considered out-of-band.",
+		Default:     1,
```

Comment:
Should we raise this a bit (maybe 5s?) since we know that the default is too low in at least one case? The issue reporter actually looked at the passwords and noticed the problem, but it might be more widespread since I think things would still work fine (i.e. you wouldn't notice) even if the rotations are really frequent.

Reply:
Maybe. I was torn about it. I didn't change the default for backwards-compatibility.

Hmmm. It would be harder to diagnose somebody with fast, automated out-of-band Active Directory password rotations. However, I do think that would be rare. 

I will bump it to 5 seconds and add a troubleshooting section to the README.

Reply:
Actually, I'll skip "troubleshooting" because boosting the logging covers it.